prometheus.go

```diff
@@ -114,25 +114,25 @@ type group struct {
 
 var (
 	groups = []group{
-		group{name: "backend", prefixes: []string{
+		{name: "backend", prefixes: []string{
```

Comment:
what auto formatter are you using to remove the struct name here? i have never seen the official go fmt do this? I think this is an unnecessary change.

Reply:
This is the result of `gofmt -s`, it "simplifies" code. Defining the struct type here is redundant.